composer.json

```diff
@@ -27,7 +27,8 @@
     "amphp/byte-stream": "^1.5.1",
     "amphp/file": "^0.3.3",
     "amphp/socket": "^0.10.11",
-    "prooph/event-store": "dev-master",
+    "prooph/event-store": "dev-async",
```

Comment:
switch back to dev-master, once https://github.com/prooph/event-store/pull/358 is merged